SpatialGDK/Source/SpatialGDK/Private/SpatialGDKSettings.cpp

```diff
@@ -187,6 +187,13 @@ void USpatialGDKSettings::PostEditChangeProperty(struct FPropertyChangedEvent& P
 	{
 		UpdateServicesRegionFile();
 	}
+	else if (Name == GET_MEMBER_NAME_CHECKED(USpatialGDKSettings, EntityPoolInitialReservationCount))
+	{
+		if (EntityPoolInitialReservationCount > 10000)
```

Comment:
One thing I would like to check before this ticket gets done is if the Runtime still imposes a limit on the number of Entity IDs you can reserve. Could you please just do a test by trying to reserve > 10,000 and seeing if it errors in the runtime logs please :D

Reply:
The Runtime does not impose this limit anymore, so closing this PR.